Tests/PhpUnit/TestCase.php

```diff
@@ -70,10 +70,9 @@ protected function getResource($resourceName)
     }
 
     /**
-     * @param $messageCatalogue
```

Comment:
Why removing and not replacing this phpdoc ?

Reply:
It's the eternal debate about "Do we need to add phpdoc when params are typehinted"